core/src/main/java/org/jdbi/v3/core/annotation/JdbiProperty.java

```diff
@@ -21,16 +21,22 @@
 import org.jdbi.v3.meta.Beta;
 
 /**
- * Configure reflective bean and pojo mapping to skip a property.
+ * Configure reflective bean and pojo property.
  */
 @Beta
 @Target({ElementType.METHOD, ElementType.FIELD})
 @Retention(RetentionPolicy.RUNTIME)
-public @interface Unmappable {
+public @interface JdbiProperty {
     /**
-     * Returns true if the property is unmappable. It will be skipped when read from the database into a pojo.
+     * Returns true if the property is mapped in a result. It will be read from the database into a pojo.
      *
-     * @return True if the property is unmappable. It will be skipped when read from the database into a pojo.
+     * @return true if the property is unmappable
      */
-    boolean value() default true;
+    boolean map() default true;
+    /**
+     * Returns true if the property is bound as an argument. Pojo property will be bound as an argument.
```

Comment:
it also applies to fields (for kotlin and the field mapper). Maybe have a more neutral explanation?